BoxOffice/Model/NetworkService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  NetworkService.swift
+//  BoxOffice
+//
+//  Created by Idinaloq, MARY on 2023/07/27.
+//
+
+import Foundation
+
+final class NetworkService {
+    typealias NetworkResult = (Result<Data, NetworkError>) -> Void
```

Comment:
typealias도 접근제한자를 붙일 수 있습니다.

Reply:
Result 👍

Reply:
typealias에 private 접근제한자를 붙이려고 시도해보았으나, fetchData 메서드에서 다음과 같은 오류가 발생하였습니다.
`Method must be declared private because its parameter uses a private type`
이는 DailyBoxOfficeViewController에서 `NetworkResult` 타입의 completion handler를 매개변수로 가진 `fetchData` 메서드를 호출하기 때문에 나타나는 것 같습니다.
따라서, 해당 수정사항은 반영하지 못하였습니다.